game/scripts/vscripts/components/musicplayer/index.lua

```diff
@@ -0,0 +1,8 @@
+--[[
+
+musicplayercontrol
+
+Player can start and pause a song using a panorama-button.
+
+]]
+require('components/musicplayer/musicplayercontrol')
```

Comment:
Ok, thank you. I switched to atom.io (from notepad++) and forgot to enable editorconf.